ignite/templates/app/files-minimal/app/app.go.plush

```diff
@@ -89,27 +83,18 @@ func init() {
 func AppConfig() depinject.Config {
 	return depinject.Configs(
 		appConfig,
-		// Alternatively, load the app config from a YAML file.
-		// appconfig.LoadYAML(AppConfigYAML),
-		depinject.Supply(
-			// supply custom module basics
-			map[string]module.AppModuleBasic{
-				genutiltypes.ModuleName: genutil.NewAppModuleBasic(genutiltypes.DefaultMessageValidator),
-				// this line is used by starport scaffolding # stargate/appConfig/moduleBasic
```

Comment:
can we remove this placeholder from the rest of the code?

Reply:
looks like it wasn't existing, so nothing to here actually.